### ISSUE TITLE
Remove donor features; it's a waste of money

### ISSUE THREAD

**Issue by rares3968** [LyricWulf/Embers#168]
I know, many of you are annoyed by this feature, it just wants to make me move to a open source midi vizualiser (here on github).
It's the donor features. Seriously? 20$ MONTHLY just to remove a watermark and render videos?
Also to edit the layout you need to pay. To edit the FUCKING SETTINGS you need to pay. To do anything you need to pay.
What you're doing with this is instead of getting money, you're annoying ALMOST ALL of the users and making them want to switch to another software.
I think that the donations should have no perks, or a Discord role and earlier access to beta versions.
Even to get on the creator showcase you need to pay $20 MONTHLY. 
I suggest to remove the paywall completely, but leave the donations for beta/alpha versions.

**Comment by lyricwulf** [comment]
Yes, I am soon going to make this project open source (hopefully within the month) and remove all software paywalls, only smaller fee for cloud-based features. Hopefully this will pave the way for Embers and the entire piano creator community.

Thank you for your feedback!
Lyric

**Comment by rares3968** [comment]
> Yes, I am soon going to make this project open source (hopefully within the month) and remove all software paywalls, only smaller fee for cloud-based features. Hopefully this will pave the way for Embers and the entire piano creator community.
> 
> Thank you for your feedback! Lyric

Great to hear that! I always loved open source apps and even made my own. Of course, in the future I will donate some money to you for making this awesome piece of software, because you deserve it!

**Comment by TheDamnedScientist** [comment]
Any update on this? @lyricwulf

**Comment by TheDamnedScientist** [comment]
> Yes, I am soon going to make this project open source (hopefully within the month) and remove all software paywalls, only smaller fee for cloud-based features. Hopefully this will pave the way for Embers and the entire piano creator community.
> 
> Thank you for your feedback! Lyric

Any update on this? @lyricwulf

**Comment by CowboyBepBop** [comment]
Not yet but it is still planned, other projects are taking time so work on Embers is slow.
An announcement will be made on the discord once that modification rolls out.

**Comment by JojoplayzMC** [comment]
Saying they should opensource it or remove the paywalls is like saying they should work for free. I do think that video rendering shouldn't cost, but honestly, watermark removal should usually cost.

**Comment by gabers-git** [comment]
> Yes, I am soon going to make this project open source (hopefully within the month) and remove all software paywalls, only smaller fee for cloud-based features. Hopefully this will pave the way for Embers and the entire piano creator community.
> 
> Thank you for your feedback! Lyric

What happened to this? @lyricwulf

**Comment by xilconeX** [comment]
> Yes, I am soon going to make this project open source (hopefully within the month) and remove all software paywalls, only smaller fee for cloud-based features. Hopefully this will pave the way for Embers and the entire piano creator community.
> 
> Thank you for your feedback! Lyric

@lyricwulf it's been 3 years, when are you gonna do it
